src/components/CommentCard.jsx

```diff
@@ -1,12 +1,43 @@
+import { useContext } from "react";
+import { UserContext } from "./context/UserProvider";
+import dateFormat from "dateformat";
+import { deleteComment } from "../api";
+
 const CommentCard = ({ comment }) => {
+  const { loggedInUser, setLoggedInUser } = useContext(UserContext);
+
+  const comment_id = comment.comment_id;
+  console.log(comment_id, "cooment id");
+  const refreshPage = () => {
+    location.href = location.href;
+  };
+
+  const deleteHandler = (comment_id) => {
+    deleteComment(comment_id).then(() => {
+      alert("Comment deleted.");
+      refreshPage();
```

Comment:
Refreshing the page is something I can understand doing here. However, I advise instead of refreshing the whole page - instead you could refetch the comments list. That means less API calls are being made and it is less disruptive to the user experience 👍